finPath/components/AppFooter.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <v-footer class="bg-grey-lighten-1">
+        <v-row justify="center" no-gutters>
+            <v-btn v-for="link in links" :key="link" class="mx-2" color="white" rounded="xl" variant="text">
+                {{ link }}
+            </v-btn>
+            <v-col class="text-center mt-4" cols="12">
+                {{ new Date().getFullYear() }} — <strong>Vuetify</strong>
+            </v-col>
+        </v-row>
+    </v-footer>
+</template>
+
+<script>
```

Comment:
Use vue 3 syntax rather than vue 2 syntax.

Reply:
Can you share some examples